com.ugetdm.uGet.yaml

```diff
@@ -0,0 +1,38 @@
+app-id: com.ugetdm.uGet
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: uget-gtk
+rename-icon: uget-icon
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  # Audio access
+  # for sound notifications
+  - --socket=pulseaudio
+  # Filesystem access
+  - --filesystem=xdg-download
+  # Network access
+  - --share=network
+  # Dconf access
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
```

Comment:
Why?

Reply:
Required for DConf?

Reply:
Did you see the first line in https://docs.flatpak.org/en/latest/sandbox-permissions.html#dconf-access ?

Reply:
I didn't, but I'd need help with the new conf lines and testing

Reply:
You can just remove those 4 lines and everything should work (check).

The rest of the doc (migration) is for applications that existed prior to that runtime/glib/portal.

Reply:
Thanks, removed